docs/middleware/android-sdk.md

```diff
@@ -1,190 +1,126 @@
+   FINBOX_RM_VERSION=<DC_SDK_VERSION>
+   FINBOX_RM_ARTIFACT=pfm-release
+   ```
+
+## Handle Permissions
+
+The Runtime permissions needs to handled by the developer when calling the helper methods. Based on the permissions available, the SDK intelligently syncs the alternate data.
 
-Then add the meta-data tag to application element.
+Below are the list of Runtime permissions the sdk adds to the application Manifest, if Manifest Merger is enabled:
 
 ```xml
-<meta-data
-    android:name="in.finbox.CLIENT_KEY_LENDING"
-    android:value="@string/finbox_client_key" />
+<uses-permission android:name="android.permission.READ_PHONE_STATE" />
+<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+<uses-permission android:name="android.permission.READ_CALENDAR" />
+<uses-permission android:name="android.permission.READ_SMS" />
+<uses-permission android:name="android.permission.RECEIVE_SMS" />
+<uses-permission android:name="android.permission.READ_CALL_LOG" />
+<uses-permission android:name="android.permission.READ_CONTACTS" />
+<uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+<uses-permission android:name="android.permission.GET_ACCOUNTS" />
+<uses-permission
+    android:name="android.permission.PACKAGE_USAGE_STATS"
+    tools:ignore="ProtectedPermissions" />
 ```
 
-## Getting Configuration ID
-This can be obtained for a given user, by calling the following API
-
-::: tip Endpoint
-POST **https://lending.finbox.in/v1/sourcing/get_configuration/**
-:::
-This API requires an API Key shared by FinBox team to be passed in `X-API-KEY` field in header.
-
-It receives following **request body** in `application/json` content type:
-```json
-{
-    "mobile": "XXXXXXXXXX",
-    "skip_modules": {
-        "onboarding": true,
-        "kyc": false
-    }
-}
-```
-The above request indicates to skip the onboarding steps, but ask for kyc step for the user of given mobile number.
-
-:::warning IMPORTANT
-Right now only onboarding and kyc modules can be skipped, rest modules are compulsory.
-:::
-
-On successful run, it will give a configuration id in `data` field in response as follows:
-```json
-{
-    "status": true,
-    "error": "",
-    "data": "<CONFIGURATION_ID>"
-}
-```
+To remove the unused permissions, add a `remove` rule to that permission as shown below:
 
-## Initializing SDK
-Once API key is added. We need to initialize the SDK. Before initializing the SDK configuration ID obtained by calling `get_configuration` API needs to be passed. Based on this ID the modules and the flow will be set in the SDK. 
+```xml
+<uses-permission
+    android:name="android.permission.READ_CALL_LOG"
+    tools:node="remove" />
+```
 
-:::danger NOTE
-The configuration ID is mandatory. If the ID passed is invalid the SDK will not start.
-:::
+In case the Manifest merger is not enabled add the above-specified permissions manually.
```

Comment:
Put this up as a note. Instead of simple text.

```
:::tip NOTE
In case the Manifest merger is not enabled add the above-specified permissions manually.
:::
```